src/scss/components/choice-tile/mixins/_hxChoiceControl.scss

```diff
@@ -0,0 +1,51 @@
+@import "vars";
+
+// ===== Mixins for <choice-tile-control> states =====
+
+// ========== Default ==========
+@mixin hxChoiceTile($pseudo-state: null) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow;
+  }
+  @else if $pseudo-state == hover {
+    background-color: $cyan-50;
+    border-color: $cyan-500;
+  }
+}
+
+// ========== SUBDUED ==========
+@mixin hxChoiceTile--subdued($pseudo-state: null) {
+  @if $pseudo-state == hover {
+    opacity: 1;
+  }
+  @else {
+    opacity: 0.7;
+  }
+}
+
+// ========== Invalid ==========
+@mixin hxChoiceTile--invalid($pseudo-state: null) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow-invalid;
+  }
+  @else if $pseudo-state == hover {
+    background-color: $red-100;
+    border-color: $red-900;
+    color: $gray-900;
+  }
+}
+
+// ========== Disabled ==========
+@mixin hxChoiceTile--disabled($pseudo-state: null) {
```

Comment:
Remove `$pseudo-state` argument. All disabled states have the same styles.

```scss
@mixin hxChoiceTile--disabled { ... }
```

Reply:
Changes has been done.